docs/tasksDir/pod-security-admission-with-kubewarden.md

```diff
@@ -113,7 +114,11 @@ kubectl get deploy -n my-namespace nginx-deployment -o json | jq ".status.condit
 }
 ```
 
-The user can fix this issue removing the `runAsUser` from the container definition:
+You can fix this by removing the `runAsUser: 0` from the container definition:
+
+<details>
+
+<summary><code>kubectl</code> command configuring a resource without <code>runAsUser: 0</code></summary>
 
 ```console
```

Comment:
This is not a requirement to merge this PR. But I think we should use `shell` syntax as the other code blocks. It's visually more pleasant and easy to read.